webpack.config.js

```diff
@@ -1,7 +1,7 @@
 var path = require('path');
 
 module.exports = options => ({
-  entry: './src/index.js',
+  entry: './lib/index.js',
```

Comment:
Updated entry point from `src` to `lib` – build TS to JS first, then use webpack to build the bundle

Reply:
You can also use https://github.com/TypeStrong/ts-loader but I'm cool with the current setup if you don't see an advantage to using ts-loader.